pkg/kv/kvserver/batcheval/cmd_add_sstable.go

```diff
@@ -59,6 +59,12 @@ func declareKeysAddSSTable(
 		// span will be tightened during evaluation.
 		l, r := rangeTombstonePeekBounds(args.Key, args.EndKey, rs.GetStartKey().AsRawKey(), nil)
 		latchSpans.AddMVCC(spanset.SpanReadOnly, roachpb.Span{Key: l, EndKey: r}, header.Timestamp)
+
+		// Obtain a read only lock on range key stats update key to serialize with
+		// range tombstone GC requests.
+		latchSpans.AddNonMVCC(spanset.SpanReadOnly, roachpb.Span{
+			Key: keys.MakeRangeIDPrefixBuf(rs.GetRangeID()).RangeTombstoneStatsUpdateKey(),
```

Comment:
Let's add a helper for this, similarly to e.g. `keys.RaftTruncatedStateKey(rangeID)`.